interchange/README.md

```diff
@@ -1,6 +1,6 @@
 # FPGA Interchange Format for RapidWright
 
-The FPGA interchange format allows designs from other tools such as [DREAMPlaceFPGA](https://github.com/rachelselinar/DREAMPlaceFPGA) to read and write placed and routed designs with RapidWright.  This interface depends on [Cap'n Proto](https://capnproto.org/index.html) for serialization and you'll need to [install](https://capnproto.org/install.html) it with the [Java plugin](https://dwrensha.github.io/capnproto-java/index.html).
+The [FPGA Interchange Format](http://www.rapidwright.io/docs/FPGA_Interchange_Format.html) allows designs from other tools such as [DREAMPlaceFPGA](https://github.com/rachelselinar/DREAMPlaceFPGA) to read and write placed and routed designs with RapidWright.  This interface depends on [Cap'n Proto](https://capnproto.org/index.html) for serialization and you'll need to [install](https://capnproto.org/install.html) it with the [Java plugin](https://dwrensha.github.io/capnproto-java/index.html).
```

Comment:
I think this sounds a little clearer, please feel free to edit further if I'm not quite capturing what you want to convey:
```suggestion
The [FPGA Interchange Format](http://www.rapidwright.io/docs/FPGA_Interchange_Format.html) provides an interface for other tools such as [DREAMPlaceFPGA](https://github.com/rachelselinar/DREAMPlaceFPGA) to import and export placed and routed designs through RapidWright and into Vivado.  This interface depends on [Cap'n Proto](https://capnproto.org/index.html) for serialization and you'll need to [install](https://capnproto.org/install.html) it with the [Java plugin](https://dwrensha.github.io/capnproto-java/index.html).
```